tests/textnote_tests.ts

```diff
@@ -7,9 +7,9 @@ import { TestOptions, VexFlowTests } from './vexflow_test_helpers';
 
 import { Crescendo } from '../src/crescendo';
 import { Flow } from '../src/flow';
-import { Font } from '../src/font';
 import { Note } from '../src/note';
 import { Stave } from '../src/stave';
+import { Tables } from '../src/tables';
```

Comment:
Are we suggesting developers to use Tables directly? I think in the past, we have considered it more of an internal class.

Reply:
Done